client/src/domain/Community/CommunityListItem.css

```diff
@@ -0,0 +1,134 @@
+main section.Community {
```

Comment:
I see you are selecting always `main elemetType.ClassName` why don't you  just select the `.ClassName`? That should be enough

Reply:
It's on my list - I'm gonna switch over to SCSS anyways, so that will resolve itself than (given there'll be enough time remaining for refactoring tasks like that ... i.e.: The end Is near- uah!)